client/galaxy/scripts/mvc/history/options-menu.js

```diff
@@ -172,16 +172,6 @@ var menu = [
         html: _l("Export History to File"),
         href: "history/export_archive?preview=True",
         anon: true
-    },
-
-    {
-        html: _l("Other Actions"),
-        header: true
-    },
-    {
-        html: _l("Import from File"),
```

Comment:
What about putting it together with the previous section, to be renamed from `Downloads` to `Import/Export` ?

Reply:
I still think it does not belong in the current history 'context' menu at all.

Reply:
We have "Create New" under the "Current History" header...

Reply:
That should imo go away too, along with other options.

Reply:
Thanks for #7565 ! IMO import and export should be together, but it's a -0.